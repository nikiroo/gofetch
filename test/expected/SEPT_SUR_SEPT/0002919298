             EMMANUELLE BÉART A ÉPOUSÉ FRÉDÉRIC CHAUDIER           

  [People] 26-09-2018 14:56

  o Reference: 0002919298
  o News link: http://s.7s7.be/3479970
  o Source link: 


    Jeanne Poma.
    
    26/09/18  -  14h56  Source: Voici, Purepeople    Emmanuel Béart
    et son époux © photo news.
    
    Les  noces  ont duré trois jours entre Paris et Gassin, dans le
    Var.
    
    Ce  vendredi  21  septembre, l'actrice de "Manon des sources" a
    épousé  son  compagnon  Frédéric  Chaudier, à la mairie du 6ème
    arrondissement  de  Paris.  Lors  de  la  cérémonie,  les trois
    enfants  d'Emmanuelle Béart étaient présents, ainsi que sa sœur
    et des amis proches.
    
    Le  lendemain,  tous  les invités attendaient les époux avec un
    ballon  rouge, couleur de la passion, sur le parvis de l'église
    Notre-Dame   de   l'Ascension,   à   Gassin.  L'endroit  a  été
    soigneusement   sélectionné   car   c'est   dans   cette  ville
    qu'Emmanuelle Béart est née, en 1963.
    
    Ce  sont  les deux frères de l'actrice qui ont eu le plaisir de
    la  conduire  à l'autel. La mariée portait une robe blanc cassé
    avec  un  dos  nu,  tandis que le marié était vêtu d'un costume
    trois pièces et cravate parme.
    
    À  bord  d'une  Méhari,  le couple est parti vers Pampelonne, à
    Ramatuelle,  où  une réception a eu lieu dans un restaurant, la
    Cabane  Bambou,  situé  sur la plage. Le lendemain, après avoir
    fait  la  fête  toute  la  nuit, les mariés et leurs invités se
    sont  retrouvés  pour  partager un dernier repas ensemble, dans
    la maison familiale.
    
    Emmanuelle  Béart  et son réalisateur de documentaires s'aiment
    depuis 2011.
    
    [1]Wedding day #cabanebambou #pampelonne #ramatuelle
    
    476  Likes,  8  Comments  -  Cabane  Bambou  (@cabanebambou) on
    Instagram: "Wedding day #cabanebambou #pampelonne #ramatuelle"
    
    
    
    [1] https://www.instagram.com/p/BnyMHbFB0CJ/?taken-by=cabaneba-
    mbou



