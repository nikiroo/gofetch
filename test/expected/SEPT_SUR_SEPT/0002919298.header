0Emmanuelle Béart a épousé Frédéric Chaudier	null/SEPT_SUR_SEPT/0002919298		70
i  [People] 26-09-2018 14:56
i
i    Les  noces  ont duré trois jours entre Paris et Gassin, dans le
i    Var.  Ce  vendredi  21  septembre,  l%27actrice de %27Manon des
i    sources%27 a épousé son ...
i
