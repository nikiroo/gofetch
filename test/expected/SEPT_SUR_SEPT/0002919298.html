<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h1>Emmanuelle Béart a épousé Frédéric Chaudier</h1>
	<div class='details'>([People] 26-09-2018 14:56)</div>
	<br/>
    <ul>
        <li>News link: <a href='http://s.7s7.be/3479970'>http://s.7s7.be/3479970</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Jeanne Poma.<br/><br/>26/09/18 - 14h56 Source: Voici, Purepeople    Emmanuel Béart et son époux © photo news.<br/><br/>Les noces ont duré trois jours entre Paris et Gassin, dans le Var.<br/><br/>Ce vendredi 21 septembre, l&#x27;actrice de &quot;Manon des sources&quot; a épousé son compagnon Frédéric Chaudier, à la mairie du 6ème arrondissement de Paris. Lors de la cérémonie, les trois enfants d&#x27;Emmanuelle Béart étaient présents, ainsi que sa sœur et des amis proches.<br/><br/>Le lendemain, tous les invités attendaient les époux avec un ballon rouge, couleur de la passion, sur le parvis de l&#x27;église Notre-Dame de l&#x27;Ascension, à Gassin. L&#x27;endroit a été soigneusement sélectionné car c&#x27;est dans cette ville qu&#x27;Emmanuelle Béart est née, en 1963.<br/><br/>Ce sont les deux frères de l&#x27;actrice qui ont eu le plaisir de la conduire à l&#x27;autel. La mariée portait une robe blanc cassé avec un dos nu, tandis que le marié était vêtu d&#x27;un costume trois pièces et cravate parme.<br/><br/>À bord d&#x27;une Méhari, le couple est parti vers Pampelonne, à Ramatuelle, où une réception a eu lieu dans un restaurant, la Cabane Bambou, situé sur la plage. Le lendemain, après avoir fait la fête toute la nuit, les mariés et leurs invités se sont retrouvés pour partager un dernier repas ensemble, dans la maison familiale.<br/><br/>Emmanuelle Béart et son réalisateur de documentaires s&#x27;aiment depuis 2011.<br/><br/>[1]Wedding day #cabanebambou #pampelonne #ramatuelle<br/><br/>476 Likes, 8 Comments - Cabane Bambou (@cabanebambou) on Instagram: &quot;Wedding day #cabanebambou #pampelonne #ramatuelle&quot;<br/><br/><br/><br/>[1] https://www.instagram.com/p/BnyMHbFB0CJ/?taken-by=cabanebambou
	</div>
<hr/>
</div>
</body>
